apis/extension/elastic_quota.go

```diff
@@ -27,17 +27,20 @@ import (
 
 // RootQuotaName means quotaTree's root\head.
 const (
-	SystemQuotaName        = "koordinator-system-quota"
-	RootQuotaName          = "koordinator-root-quota"
-	DefaultQuotaName       = "koordinator-default-quota"
-	QuotaKoordinatorPrefix = "quota.scheduling.koordinator.sh"
-	LabelQuotaIsParent     = QuotaKoordinatorPrefix + "/is-parent"
-	LabelQuotaParent       = QuotaKoordinatorPrefix + "/parent"
-	LabelAllowLentResource = QuotaKoordinatorPrefix + "/allow-lent-resource"
-	LabelQuotaName         = QuotaKoordinatorPrefix + "/name"
-	AnnotationSharedWeight = QuotaKoordinatorPrefix + "/shared-weight"
-	AnnotationRuntime      = QuotaKoordinatorPrefix + "/runtime"
-	AnnotationRequest      = QuotaKoordinatorPrefix + "/request"
+	SystemQuotaName           = "koordinator-system-quota"
+	RootQuotaName             = "koordinator-root-quota"
+	DefaultQuotaName          = "koordinator-default-quota"
+	QuotaKoordinatorPrefix    = "quota.scheduling.koordinator.sh"
+	LabelQuotaIsParent        = QuotaKoordinatorPrefix + "/is-parent"
+	LabelQuotaParent          = QuotaKoordinatorPrefix + "/parent"
+	LabelAllowLentResource    = QuotaKoordinatorPrefix + "/allow-lent-resource"
+	LabelQuotaName            = QuotaKoordinatorPrefix + "/name"
+	AnnotationSharedWeight    = QuotaKoordinatorPrefix + "/shared-weight"
+	AnnotationRuntime         = QuotaKoordinatorPrefix + "/runtime"
+	AnnotationRequest         = QuotaKoordinatorPrefix + "/request"
+	AnnotationQuotaNamespaces = QuotaKoordinatorPrefix + "/namespaces"
+
+	IndexQuotaNamespaces = "annotation.namespaces"
```

Comment:
We cannot define the index name here. The file is used to declare the extension APIs.